includes/event/event-fields.php

```diff
@@ -113,14 +113,14 @@ function event_organizer_callback($post)
 {
     wp_nonce_field(basename(__FILE__), 'event_organizer_nonce');
     $organizers = get_posts(array(
-        'post_type' => 'organizer',
+        'post_type' => 'organisationer',
```

Comment:
A good practice is to keep all the codes in English and use the common i18 approaches to translate them in the target language(s) (i.e .PO files which are editable via [Poedit tool](https://poedit.net/) and be compiled in .MO files)
You may look up functions with __ prefix 
ex: __('Event venue', 'agenda-core')